op-node/rollup/derive/engine_controller.go

```diff
@@ -298,6 +322,64 @@ func (e *EngineController) InsertUnsafePayload(ctx context.Context, payload *eth
 	return nil
 }
 
+// tryBackupUnsafeReorg attempts to reorg(restore) unsafe head to backupUnsafeHead.
```

Comment:
nit: its a public function but the godoc shows its not (lowercased)

Reply:
Fixed at https://github.com/ethereum-optimism/optimism/pull/8925/commits/f363524c1d40c4566bf20e8d6d23395580ac8fb2.

Reply:
Fix comments at https://github.com/ethereum-optimism/optimism/pull/8925/commits/6a5296dc5f47a943638f664d29aef73763402333.